### PR TITLE
[DOC] Decoding and introduction documentation enhancements

### PR DESCRIPTION
**Pull request by NicolasGensollen**
This PR proposes some small enhancements to the user guide (introduction and decoding) by:

- fixing some broken links and adding a few new ones
- fixing some formatting issues
- adding dummy options for the estimators
- converting hard-coded scientific references to Bibtex (they will be much easier to maintain and will appear in the general bibliography).

Still WIP...

### CODE REVIEW

File: doc/decoding/going_further.rst [hunk 1]
```diff
@@ -53,35 +49,28 @@ of cross-validation.
...
-* use a different cross-validation scheme, for example LeaveOneGroupOut()
+* use a different cross-validation scheme, for example :class:`sklearn.model_selection.LeaveOneGroupOut`.
 
-* speed up the computation by using n_jobs=-1, which will spread the
-   computation equally across all processors.
+* speed up the computation by using `n_jobs=-1`, which will spread the computation equally across all processors.
 
-* use a different scoring function, as a keyword or imported from scikit-learn
-   such as `scoring='roc_auc'`
+* use a different scoring function, as a keyword or imported from scikit-learn such as `scoring='roc_auc'`.
```
Comment (bthirion) [inline doc/decoding/going_further.rst:56]: Just curious: what is the reason for this change ?

File: doc/references.bib [hunk 2]
```diff
@@ -373,6 +484,39 @@ @article{JIMURA2012544
...
+    abstract = {Summary
+The dominant view that perceptual learning is accompanied by changes in early sensory representations has recently been challenged. Here we tested the idea that perceptual learning can be accounted for by reinforcement learning involving changes in higher decision-making areas. We trained subjects on an orientation discrimination task involving feedback over 4 days, acquiring fMRI data on the first and last day. Behavioral improvements were well explained by a reinforcement learning model in which learning leads to enhanced readout of sensory information, thereby establishing noise-robust representations of decision variables. We find stimulus orientation encoded in early visual and higher cortical regions such as lateral parietal cortex and anterior cingulate cortex (ACC). However, only activity patterns in the ACC tracked changes in decision variables during learning. These results provide strong evidence for perceptual learning-related changes in higher order areas and suggest that perceptual and reward learning are based on a common neurobiological mechanism.}
+}
+
+
+@article {Kriegeskorte3863,
```
Comment (bthirion) [inline doc/references.bib:504]:
```suggestion
@article{Kriegeskorte3863,
```

File: doc/decoding/going_further.rst (hunk 1 above)
Comment (NicolasGensollen) [reply doc/decoding/going_further.rst:56]: @bthirion I just added a period at the end of the line to be consistent with lines above, and improved the rendering:

Before:

before


After:

after

File: doc/references.bib (hunk 2 above)
Comment (NicolasGensollen) [reply doc/references.bib]: Thanks!

File: doc/decoding/going_further.rst (hunk 1 above)
Comment (bthirion) [reply doc/decoding/going_further.rst:56]: OK, thx.

**Review by bthirion — APPROVED** [review]
LGTM, thx !

**Comment by NicolasGensollen** [comment]
Thanks, merging....
